.travis.yml

```diff
@@ -1,13 +1,26 @@
-sudo: false
-language: node_js
-node_js :
- - "0.12"
+# This file was adapted from work by Keith James (keithj). The original source
+# can be found as part of the wtsi-npg/data_handling project here:
+#
+#   https://github.com/wtsi-npg/data_handling
+
+sudo: required
+
+language: perl
+
+perl:
+  - "5.16"
+
+env:
+  global:
+    - DNAP_WAREHOUSE_VERSION="1.5"
+    - NPG_ML_WAREHOUSE_VERSION="37.9"
+    - NPG_SEQ_COMMON_VERSION="50.7"
+    - NPG_TRACKING_VERSION="84.10"
```

Comment:
Already changed code in install.sh accordingly. Though I liked the idea of having a single-easy access place to change tags/branches. Otherwise we need to dig in the shell scripts to test against something specific.